plugins/cost-insights/src/example/client.ts

```diff
@@ -27,20 +27,17 @@ import {
   Project,
   ProjectGrowthData,
   UnlabeledDataflowData,
-} from '../src/types';
-import {
-  ProjectGrowthAlert,
-  UnlabeledDataflowAlert,
-  KubernetesMigrationAlert,
-} from '../src/alerts';
+} from '../types';
+import { KubernetesMigrationAlert } from './alerts';
+import { ProjectGrowthAlert, UnlabeledDataflowAlert } from '../alerts';
 import {
   aggregationFor,
   changeOf,
   entityOf,
   getGroupedProducts,
   getGroupedProjects,
   trendlineOf,
-} from './utils/mockData';
+} from '../utils/mockData';
```

Comment:
Seems like mockData should go in testUtils too 🤔

Reply:
Ah, good catch. Yeah, there are some exports used only in example code that should be moved over. 

But we prob don't want to export all mockData in `/testUtils` since some of it is for internal purposes.